### PR TITLE
docs(ee/migrations) Fixing the 1.5.x to 2.1.x migration instructions

### PR DESCRIPTION
**Pull request by tyler-ball**
The 1.5.x to 2.1.x migration cannot be performed without downtime like
most upgrades can. Instead, the user must take steps to cache all
entities on the 1.5 cluster to ensure it does not attempt to modify the
database while the 2.1 cluster is being brought up.

Existing docs: https://docs.konghq.com/enterprise/2.1.x/deployment/upgrades/migrations/#migrating-from-15x-or-21x-beta-to-21x

Signed-off-by: Tyler Ball <tyler.ball@konghq.com>

### CODE REVIEW

File: app/enterprise/2.1.x/deployment/upgrades/migrations.md [hunk 1]
```diff
@@ -69,44 +69,44 @@ affect your current installation.
...
+   no additional entities need to be warmed in the cache (added to the list in
+   the configuration).
 
-4. Gradually divert traffic away from your old nodes, and redirect traffic to
+3. Download 2.1.x, and configure it to point to the same datastore as your old
+   1.5.x (or 2.1.x-beta) cluster.
+4. Run `kong migrations up` on the 2.1.x cluster.
+5. Run `kong migrations finish` on the 2.1.x cluster. From this point onward, it
+   is no longer possible to start nodes in the old 1.5 (or 2.1.x-beta) cluster
+   or access its admin API. Run this command _only_ when you are confident that
```
Comment (tyler-ball) [inline app/enterprise/2.1.x/deployment/upgrades/migrations.md:101]: I wasn't completely sure if users could no longer access the admin API on the 1.5 cluster - do changes to the 1.5 admin API get stored in the database?

Also, looking at this, it probably needs to be moved to `4.` if the `migrations up` command would interfere with the the ability of the 1.5 cluster to store admin API changes.

**Comment by tlblessing** [comment]
Direct review link: https://deploy-preview-2561--kongdocs.netlify.app/enterprise/2.2.x/deployment/upgrades/migrations/

File: app/enterprise/2.1.x/deployment/upgrades/migrations.md (hunk 1 above)
Comment (tlblessing) [reply app/enterprise/2.1.x/deployment/upgrades/migrations.md:101]: > I wasn't completely sure if users could no longer access the admin API on the 1.5 cluster - do changes to the 1.5 admin API get stored in the database?
> 
> Also, looking at this, it probably needs to be moved to `4.` if the `migrations up` command would interfere with the the ability of the 1.5 cluster to store admin API changes.

@tyler-ball thank you for the PR. I cannot answer your question; an engineer should be able to confirm and also review and test this PR. Any chance you could assist with the 2.2 to 2.3 migration docs/testing as well? @RobSerafini who can assist docs with this? cc: @GayleNeumann

**Review by mikefero — APPROVED** [review]
:rocket:

**Comment by tyler-ball** [comment]
I rebased and got a review - this is ready to merge!

**Review by HCloward — APPROVED** [review]

**Pull request merged**